x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/handle_api_errors.ts

```diff
@@ -31,12 +31,13 @@ interface Options {
   isQueued?: boolean;
 }
 
+// TODO I know our error messages from the BE are not i18n-ized but should this be?
```

Comment:
@constancecchen

Reply:
Currently none of our error messages from the BE are localized, but if this one is in `public/` it has access to `i18n`, so we should likely i18n it just in case.

Reply:
Also want to add it wouldn't be my preference to export this const. I'd just check for the static `'An unexpected error occurred'` string in the schema logic file. It's short, easy to read, and gives developers human-readable context over a variable that means nothing if they don't open another file to check it

Reply:
Oh wait, sorry, just saw we're using it in the actual logic file, not just the test. Hm, sec

Reply:
I'll i18n this